### PR TITLE
Add target audience and how to read doc to the intro tutorial

### PR DESCRIPTION
**Pull request by dstansby**
This adds some narrrative documentation the describes the purpose, target audience, and "how to read" of the introductory tutorial. All up for bikeshedding and feedback, but I think this broadly explains what we discussed at the documentation sprint.

### CODE REVIEW

File: docs/intro/index.rst [hunk 1]
```diff
@@ -1,23 +1,37 @@
...
+Each chapter of the tutorial provides a self-contained set of codes.
+
+Later chapters build on the concepts introduced in earlier chapters.
+The one exception is the final two chapters that explore Maps and TimeSeries - these are independent, but depend on all earlier chapters.
 
 .. toctree::
    :maxdepth: 2
 
    installation
    Data Acquisition <acquiring_data/index>
```
Comment (Cadair) [inline docs/intro/index.rst:30]:
```suggestion
   acquiring_data/index
```

File: docs/intro/index.rst [hunk 2]
```diff
@@ -1,23 +1,37 @@
...
+This tutorial explains the basics of these packages needed to use sunpy, but is not a comprehensive tutorial to these pacakges and references their own introductoriy tutorials where relevant.
+
+How to use this tutorial
+------------------------
+This tutorial is designed to be read from start to finish.
+You can either read it without running any code, or copy and paste the code snippets as you read.
+Each chapter of the tutorial provides a self-contained set of codes.
+
+Later chapters build on the concepts introduced in earlier chapters.
+The one exception is the final two chapters that explore Maps and TimeSeries - these are independent, but depend on all earlier chapters.
```
Comment (Cadair) [inline docs/intro/index.rst:24]:
```suggestion
The one exception is the two chapters that explore Maps and TimeSeries - these are independent, but depend on all earlier chapters.
```

**Review by Cadair — APPROVED** [review]

File: docs/intro/index.rst [hunk 3]
```diff
@@ -1,23 +1,37 @@
 .. _guide:
 
-***************
-Getting Started
-***************
+**************
+Intro tutorial
+**************
```
Comment (nabobalis) [inline docs/intro/index.rst:5]:
```suggestion
**********************
The ``sunpy`` tutorial
**********************
```

File: docs/intro/index.rst [hunk 4]
```diff
@@ -1,23 +1,37 @@
 .. _guide:
 
-***************
-Getting Started
-***************
+**************
+Intro tutorial
```
Comment (Cadair) [inline docs/intro/index.rst:4]: I am not a big fan of this title.

Maybe "sunpy tutorial" or something?

File: docs/intro/index.rst [hunk 5]
```diff
@@ -1,23 +1,37 @@
...
+**************
 
-Welcome to the user guide for the sunpy core package. sunpy is a community-
-developed, free and open-source solar data analysis environment. It is meant to
-provide the core functionality and tools to analyze solar data with Python.
-This guide provides a walkthrough of the major features in sunpy.
-For more details checkout the :ref:`reference`.
+Welcome to the introductory tutorial for the sunpy core package.
+sunpy is a community-developed, free and open-source solar data analysis environment.
```
Comment (nabobalis) [inline docs/intro/index.rst:8]:
```suggestion
Welcome to the introductory tutorial for the ``sunpy`` core package.
``sunpy`` is a community-developed, free and open-source solar data analysis environment.
```

File: docs/intro/timeseries.rst [hunk 6]
```diff
@@ -1,8 +1,8 @@
 .. _timeseries_guide:
 
-**********************
-Time series data guide
-**********************
+**********
+Timeseries
```
Comment (nabobalis) [inline docs/intro/timeseries.rst:4]: Do we want to link this to the class?

Comment (Cadair) [reply docs/intro/timeseries.rst:4]: I do not, I don't like verbatim or links in titles.

**Comment by dstansby** [comment]
👍 title updated

**Review by nabobalis — APPROVED** [review]

File: docs/intro/index.rst [hunk 7]
```diff
@@ -1,23 +1,37 @@
 .. _guide:
 
-***************
-Getting Started
-***************
+**********************
+The ``sunpy`` tutorial
```
Comment (Cadair) [inline docs/intro/index.rst:4]: I really don't like the highlighting effect of using verbatim blocks in titles (generally).

This:
title

just looks ugly to me.

```suggestion
The sunpy tutorial
```

Comment (dstansby) [reply docs/intro/index.rst]: 👍 agreed

**Review by Cadair — APPROVED** [review]

**Pull request merged**